### ISSUE TITLE
No Activity found to handle Intent { act=android.media.action.IMAGE_CAPTURE_SECURE (has extras)

### ISSUE THREAD

**Issue by amaydiam** [jkwiecien/EasyImage#6]
with your sample, i have error :
``11-07 22:15:07.773 28618-28618/pl.aprilapps.easyphotopicker.sample W/System.err: android.content.ActivityNotFoundException: No Activity found to handle Intent { act=android.media.action.IMAGE_CAPTURE_SECURE (has extras) }
11-07 22:15:07.774 28618-28618/pl.aprilapps.easyphotopicker.sample W/System.err:     at android.app.Instrumentation.checkStartActivityResult(Instrumentation.java:1638)
11-07 22:15:07.774 28618-28618/pl.aprilapps.easyphotopicker.sample W/System.err:     at android.app.Instrumentation.execStartActivity(Instrumentation.java:1430)
11-07 22:15:07.774 28618-28618/pl.aprilapps.easyphotopicker.sample W/System.err:     at android.app.Activity.startActivityForResult(Activity.java:3470)
11-07 22:15:07.774 28618-28618/pl.aprilapps.easyphotopicker.sample W/System.err:     at android.app.Activity.startActivityForResult(Activity.java:3426)
11-07 22:15:07.774 28618-28618/pl.aprilapps.easyphotopicker.sample W/System.err:     at android.support.v4.app.FragmentActivity.startActivityForResult(FragmentActivity.java:784)
11-07 22:15:07.774 28618-28618/pl.aprilapps.easyphotopicker.sample W/System.err:     at pl.aprilapps.easyphotopicker.EasyImage.openCamera(EasyImage.java:71)
11-07 22:15:07.774 28618-28618/pl.aprilapps.easyphotopicker.sample W/System.err:     at pl.aprilapps.easyphotopicker.sample.MainActivity.onTakePhotoClicked(MainActivity.java:43)
11-07 22:15:07.774 28618-28618/pl.aprilapps.easyphotopicker.sample W/System.err:     at pl.aprilapps.easyphotopicker.sample.MainActivity$$ViewBinder$1.doClick(MainActivity$$ViewBinder.java:19)
11-07 22:15:07.774 28618-28618/pl.aprilapps.easyphotopicker.sample W/System.err:     at butterknife.internal.DebouncingOnClickListener.onClick(DebouncingOnClickListener.java:22)
11-07 22:15:07.774 28618-28618/pl.aprilapps.easyphotopicker.sample W/System.err:     at android.view.View.performClick(View.java:4561)
11-07 22:15:07.774 28618-28618/pl.aprilapps.easyphotopicker.sample W/System.err:     at android.view.View$PerformClick.run(View.java:18941)
11-07 22:15:07.774 28618-28618/pl.aprilapps.easyphotopicker.sample W/System.err:     at android.os.Handler.handleCallback(Handler.java:808)
11-07 22:15:07.775 28618-28618/pl.aprilapps.easyphotopicker.sample W/System.err:     at android.os.Handler.dispatchMessage(Handler.java:103)
11-07 22:15:07.775 28618-28618/pl.aprilapps.easyphotopicker.sample W/System.err:     at android.os.Looper.loop(Looper.java:193)
11-07 22:15:07.775 28618-28618/pl.aprilapps.easyphotopicker.sample W/System.err:     at android.app.ActivityThread.main(ActivityThread.java:5395)
11-07 22:15:07.775 28618-28618/pl.aprilapps.easyphotopicker.sample W/System.err:     at java.lang.reflect.Method.invokeNative(Native Method)
11-07 22:15:07.775 28618-28618/pl.aprilapps.easyphotopicker.sample W/System.err:     at java.lang.reflect.Method.invoke(Method.java:515)
11-07 22:15:07.775 28618-28618/pl.aprilapps.easyphotopicker.sample W/System.err:     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:837)
11-07 22:15:07.775 28618-28618/pl.aprilapps.easyphotopicker.sample W/System.err:     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:653)
11-07 22:15:07.775 28618-28618/pl.aprilapps.easyphotopicker.sample W/System.err:     at dalvik.system.NativeStart.main(Native Method)``

on  lenovo, android API 19.
please fix it

**Comment by ano-labs** [comment]
i have this issue too, but if you got other camera app this sample worked. Please fix it btw

**Comment by jkwiecien** [comment]
I don't really have anything to test it on, you'd have to help me guys.

Please go to EasyImage class and change this piece:
```java
Intent intent;
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1) {
            intent = new Intent(MediaStore.ACTION_IMAGE_CAPTURE_SECURE);
        } else {
            intent = new Intent(MediaStore.ACTION_IMAGE_CAPTURE);
        }
```
to
```java
Intent intent = new Intent(MediaStore.ACTION_IMAGE_CAPTURE);
```

That won't help, rather create another issue which was already reported but will give me some clue.

**Comment by jkwiecien** [comment]
I think I got it. Expect fix.

**Comment by jkwiecien** [comment]
Hopefully fixed in https://github.com/jkwiecien/EasyImage/pull/9

**Comment by shimiuh** [comment]
I am getting the same error  after fix#9 do we have a fix?
